builder.config.json

```diff
@@ -37,91 +37,91 @@
       }
     ],
     "blockClasses": {
-      "ConsoleIO\\Print": {
+      "components/interaction-designer/block-types/ConsoleIO_PrintBlock": {
```

Comment:
@bzabos , wepback seems to require a minimun static path prefix to accept dynamic variable (eg "`@/`"), https://github.com/FLOIP/flow-builder/pull/37/files#diff-4778731e3debd774a6c14025179b9bfb34aae3d45fb42448205377ca6475503dR219

So I didn't add `@` here, let me know if this works as you expected